hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BinarySerializer.java

```diff
@@ -213,8 +213,9 @@ protected void parseProperties(BytesBuffer buffer, HugeElement owner) {
         buffer.writeId(edge.ownerVertex().id());
         buffer.write(edge.type().code());
         buffer.writeId(edge.schemaLabel().id());
-        buffer.writeString(edge.name()); // TODO: write if need
+        buffer.writeStringWithoutLength(edge.name());
```

Comment:
change to writeStringWithEnding() to write ending symbol 0xFF
and don't write length at line 218 since we can find by 0xFF